aio/aio-proxy/aio_proxy/search/helpers/color.py

```diff
@@ -2,12 +2,15 @@
 import logging
 from urllib.request import urlopen
 
-from aio_proxy.response.helpers import COLOR_URL
+from aio_proxy.response.helpers import COLOR_URL, is_dev_env
 
 logging.basicConfig(level=logging.INFO)
 
 
 def get_current_color(color_url):
+    if is_dev_env() is False:
+        return "reader"
```

Comment:
I don't quite understand the "reader" logic. What is the point of this string ?

Reply:
The github workflow is not compatible with the "siren-reader" alias since the dev server has not been migrated yet.

Reply:
I see.